packages/common/src/components/Preview/index.tsx

```diff
@@ -507,6 +512,14 @@ class BasePreview extends React.Component<Props, State> {
     });
   };
 
+  refreshHashedUrl = url => {
+    if (url.includes('#')) {
+      dispatch({
+        type: 'refresh',
+      });
+    }
```

Comment:
You could use an early return

```suggestion
    if (!url.includes('#') {
      return;
    }
    
    dispatch({ type: 'refresh' });
```

Reply:
Done